docs/taskruns.md

```diff
@@ -884,6 +884,9 @@ When you cancel a TaskRun, the running pod associated with that `TaskRun` is del
 means that the logs of the `TaskRun` are not preserved. The deletion of the `TaskRun` pod is necessary
 in order to stop `TaskRun` step containers from running.
 
+**Note: if `enable-cancel-using-entrypoint` is set to
```

Comment:
Could we rename this feature flag in terms of the user-facing impact? e.g. "keep-pod-on-cancel" or something like that. Also, it's probably worth noting this uses the downward API, and might not be suitable in environments that don't allow it (https://github.com/tektoncd/pipeline/issues/4937). This is why we introduced the "await-sidecar-readiness" flag. We should also note the default value. lastly, it's probably worth describing this as an alpha feature, and adding it to our alpha features table. https://github.com/tektoncd/pipeline/blob/main/docs/additional-configs.md#alpha-features

Reply:
1. renamed to "keep-pod-on-cancel" and add this flag to alpha-features
2. at present, it is still necessary to use the downward API to pass the cancel signal. setting `keep-pod-on-cancel` to true will make the pod use the downward api despite using the "await-sidecar-readiness" flag